index.html

```diff
@@ -69,5 +69,130 @@ <h2 class="connect">LET'S CONNECT</h2>
 				</section>
 				<span class="header-shape"></span>
 			</main>
+      <section class="work">
+        <div class="cardWork sec">
```

Comment:
- [OPTIONAL]  I think It would be better to use `<article>` tag here instead of a div to make it more semantic :recycle: